README.md

```diff
@@ -1 +1,93 @@
-# bindplane-op-action
\ No newline at end of file
+[![CI](https://github.com/observIQ/bindplane-op-action/actions/workflows/ci.yml/badge.svg)](https://github.com/observIQ/bindplane-op-action/actions/workflows/ci.yml)
+
+# bindplane-op-action
+
+The BindPlane OP action can be used to deploy configurations to your BindPlane OP
+server. It also supports exporting the OpenTelemetry configurations back to the repository.
+
+## Configuration
+
+| Parameter                     | Default    | Description                     |
+| :---------------------------- | :--------- | :------------------------------ |
+| bindplane_remote_url          | required   | The endpoint that will be used to connect to BindPalne OP. |
+| bindplane_api_key             |            | API key used to authenticate to BindPlane. Required when BindPlane multi account is enabled or when running on BindPlane Cloud |
+| bindplane_username            |            | Username used to authenticate to BindPlane. Not required if API key is set. |
+| bindplane_password            |            | Password used to authenticate to BindPlane.
+| target_branch                 | required   | The branch that the action will use when applying resources to bindplane or when writing otel configs back to the repo. |
+| destination_path              |            | Path to the file which contains the BindPlane destination resources |
+| configuration_path            |            | Path to the file which contains the BindPlane configuration resources |
+| enable_otel_config_write_back | `false`    | Whether or not the action should write the raw OpenTelemetry configurations back to the repository. | 
+| configuration_output_dir      |            | When write back is enabled, this is the path that will be written to. |
+| token                         |            | The Github token that will be used to write to the repo. Usually secrets.GITHUB_TOKEN is sufficient. Requires the `contents.write` permission. |
+
+## Usage
+
+### Export Resources
+
+To get started, you must handle exporting your existing resources to the repository. Use
+the `bindplane get` commands with the `--export` flag.
+
+```bash
+bindplane get destination -o yaml --export > destination.yaml
+bindplane get configuration -o yaml --export > configuration.yaml
```

Comment:
In the PR instructions it has plural of these file names `destinations.yaml` and `configurations.yaml` what do we want? It failed for me as I followed the PR instructions.

Reply:
Looks like you got it working. The path you export the initial resources too should match whatever you configure your action to point to, in your repo you have:
```
          destination_path: destination.yaml
          configuration_path: configuration.yaml
```

but you could have anything. My [example repo](https://github.com/jsirianni/bindplane-op-action-test) has the following structure:
```
.
├── otel
│   ├── k8s-cluster.yaml
│   ├── k8s-gateway.yaml
│   └── k8s-node.yaml
├── README.md
└── resources
    ├── configurations
    │   └── resource.yaml
    └── destinations
        └── resource.yaml
```

Where `resources/` is the top level directory for my destinations and configurations. The output dir is `otel`.